janusgraph-test/src/main/java/org/janusgraph/diskstorage/indexing/IndexProviderTest.java

```diff
@@ -950,6 +950,8 @@ public void testCustomAnalyzer() throws Exception {
 
         IndexQuery query = new IndexQuery(store, PredicateCondition.of(STRING, Cmp.EQUAL, "Tom and Jerry"));
         assertEquals(1, tx.queryStream(query).count(), query.toString());
+        query = new IndexQuery(store, PredicateCondition.of(STRING, Cmp.EQUAL, "Tom"));
```

Comment:
This test only checks the case `janusgraphPredicate == Cmp.EQUAL`. Could you please also add tests for the other two cases here: `janusgraphPredicate == Text.CONTAINS` and `janusgraphPredicate == Cmp.NOT_EQUAL`?

Reply:
Ok

I add test for this predicates and remove extra NOT_EQUAL case in converteQuery, now handled in tokenize method